common/src/main/java/io/pravega/example/tensorflow/YOLOClassifier.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.example.tensorflow;
+
+import org.apache.commons.math3.analysis.function.Sigmoid;
+import org.tensorflow.Tensor;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.PriorityQueue;
+
+
+public class YOLOClassifier {
+    private final static float OVERLAP_THRESHOLD = 0.5f;
+    private final static double anchors[] = {1.08,1.19,  3.42,4.41,  6.63,11.38,  9.42,5.11,  16.62,10.52};
+    private final static int SIZE = 13;
```

Comment:
What is SIZE? You should rename this to be more descriptive.

Reply:
It is the number of boxes that the image is broken down into for detection.